### PR TITLE
store navigationTiming marks as scaled floats

### PR DESCRIPTION
**Pull request by graphaelli**
for #708.  Amends the elasticsearch mapping template to include:
```
{
  "transaction.marks.navigationTiming": {
    "mapping": {
      "type": "scaled_float",
      "scaling_factor": 1000000
    },
    "match_mapping_type": "*",
    "path_match": "transaction.marks.navigationTiming.*"
  }
}
```

Given this event:

```
{
  "@timestamp": "2017-05-30T18:53:27.100Z",
  "transaction": {
    "marks": {
      "navigationTiming": {
        "appBeforeBootstrap": 608.9300000000001,
        "navigationStart": -21
      }
    }
  }
}
```

the current elasticsearch mapping would be:
```
{
  "marks": {
    "properties": {
      "navigationTiming": {
        "properties": {
          "appBeforeBootstrap": {
            "type": "float"
          },
          "navigationStart": {
            "type": "long"
          }
        }
      }
    }
  }
}
```

with this change, the mapping ends up:
```
{
  "marks": {
    "dynamic": "true",
    "properties": {
      "navigationTiming": {
        "dynamic": "true",
        "properties": {
          "appBeforeBootstrap": {
            "type": "scaled_float",
            "scaling_factor": 1000000
          },
          "navigationStart": {
            "type": "scaled_float",
            "scaling_factor": 1000000
          }
        }
      }
    }
  }
}
```

This is a special case update to store `navigationTiming`s.

### CODE REVIEW

**Comment by jalvz** [comment]
are we waiting for something to merge this?

**Comment by graphaelli** [comment]
integration test data - @jalvz @simitt mind checking it out?

File: processor/transaction/package_tests/TestProcessTransactionFull.approved.json [hunk 1]
```diff
@@ -119,6 +119,10 @@
                 },
                 "id": "945254c5-67a5-417e-8a4e-aa29efcbfb79",
                 "marks": {
+                    "another_mark": {
```
Comment (jalvz) [inline processor/transaction/package_tests/TestProcessTransactionFull.approved.json]: need to run approvals again, these are now called `some_long` `some_float`

Comment (graphaelli) [reply processor/transaction/package_tests/TestProcessTransactionFull.approved.json]: thanks, done

**Pull request merged**